apps/website/src/components/EditPageLink.astro

```diff
@@ -0,0 +1,33 @@
+---
+import EditIcon from '~/icons/EditIcon.astro';
+
+type Props = { page: string };
+const { page } = Astro.props;
+
+const githubEditUrl = `https://github.com/iTwin/iTwinUI-react/edit/main/apps/website/src/pages/${page}.mdx`;
+---
+
+<a href={githubEditUrl}>
```

Comment:
Open in new tab.

Reply:
Open in new tab is generally considered an accessibility pattern. Any reason you want it here?

Reply:
It would be more convenient (for me). You don't have to navigate back after changing text in github and creating PR.
I guess it's ok if it's considered an anti-pattern.